index.html

```diff
@@ -1,167 +1,278 @@
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div>
+            <ul class="social-media-icon">
+              <li><img src="/img/social icons/twiter_icon.svg" alt="" /></li>
+              <li><img src="/img/social icons/github_icon.svg" alt="" /></li>
+              <li><img src="/img/social icons/icon.svg" alt="" /></li>
+              <li><img src="/img/social icons/Linkedin_icon.svg" alt="" /></li>
+              <li><img src="/img/social icons/medium_icon.svg" alt="" /></li>
+            </ul>
+            <a href="#" class="connect-two">LET'S CONNECT</a>
+          </div>
+          <br />
+          <button class="btn-primary">Get my resume</button>
+          <div class="modal">
+            <ul class="modal-items">
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Languages
+                  <img src="/img/Icon - Chevron - Down - Larger.png" alt="" />
+                </h3>
+                <br />
+                <span class="item">
+                  <img src="/img/js.png" alt="" />
+                  <h4>Javascript</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/html5.png" alt="" />
+                  <h4>HTML</h4></span
+                >
+                <br />
+                <span class="item">
+                  <img src="/img/css3.png" alt="" />
+                  <h4>CSS</h4></span
+                >
+                <br />
+              </li>
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Frameworks <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+              <li class=".modal-item">
+                <h3 class="item-one">
+                  Skills <img src="/img/Union.png" alt="" />
+                </h3>
+              </li>
+              <hr />
+            </ul>
+          </div>
+        </div>
+      </section>
+
+      <section id="contact">
+        <div class="contact-container">
+          <div class="contact-desc">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <div class="form">
+            <form
+              id="fs-frm"
+              name="simple-contact-form"
+              accept-charset="utf-8"
+              action="https://formspree.io/f/xnqrdrre"
+              method="post"
+            >
+              <fieldset id="fs-frm-inputs">
+                <input
+                  type="text"
+                  name="name"
+                  id="full-name"
+                  required=""
+                  placeholder="Raj"
```

Comment:
[OPTIONAL] I  suggest you type a placeholder that is more informational. Instead of using your name, you can use the text `enter name here`. This despite providing required information also informs the user what is required of him